plugin/src/steps/ingest-remote-schema/diff-schemas.js

```diff
@@ -35,7 +34,7 @@ const checkIfSchemaHasChanged = async ({ traceId }) => {
   }
 
   const { data } = await fetchGraphql({
-    query: gql`
+    query: /* GraphQL */ `
```

Comment:
Good call. We should delete `utils/gql` if you didn't already. It's not really needed anywhere and doesn't do more than a comment

Reply:
Ah, nice, you did delete it!